app/controllers/users_controller.rb

```diff
@@ -116,8 +117,8 @@ def comment
 
     # Create the story.
     @user = User.find(params[:user_id])
-    story = Action.broadcast(
-      action_type: "created_profile_comment",
+    _story = Action.broadcast(
```

Comment:
I assume `_story` is not used anywhere, hence the underscore — probably cleaner to just remove the assignment